modcc/mechanism.hpp

```diff
@@ -6,7 +6,7 @@
     Abstract base class for all mechanisms
 
     This works well for the standard interface that is exported by all mechanisms,
-    i.e. nrn_jacobian(), nrn_current(), etc. The overhead of using virtual dispatch
+    i.e. nrn_jacobian(), compute_currents(), etc. The overhead of using virtual dispatch
```

Comment:
oops, we can remove `nrn_jacobian` from this comment!

Reply:
Yes, it's gone locally, as discussed on Slack.

Reply:
Done